be2-scala/src/main/resources/logback.xml

```diff
@@ -0,0 +1,14 @@
+<!--Configuration for ch.qos.logback-->
+<!--Set debug to true to show debug logs on STDOUT-->
+<configuration debug="false" scan="true" scanPeriod="15 seconds">
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+<!--Set root(general) logs level to INFO/DEBUG on STDOUT-->
+  <root level="INFO">
+    <appender-ref ref="STDOUT"/>
```

Comment:
This would be a separate PR, but if you want to take advantage of [Apache Chainsaw](https://logging.apache.org/chainsaw/2.x/), you could configure a socket appender.
It's really super helpful when debugging locally.

Reply:
👍